specs/deneb/p2p-interface.md

```diff
@@ -51,6 +52,8 @@ The specification of these changes continues in the same format as the network s
 | `MAX_REQUEST_BLOB_SIDECARS`              | `MAX_REQUEST_BLOCKS_DENEB * MAX_BLOBS_PER_BLOCK` | Maximum number of blob sidecars in a single request  |
 | `MIN_EPOCHS_FOR_BLOB_SIDECARS_REQUESTS`  | `2**12` (= 4096 epochs, ~18 days) | The minimum epoch range over which a node must serve blob sidecars  |
 | `BLOB_SIDECAR_SUBNET_COUNT`              | `6`                               | The number of blob sidecar subnets used in the gossipsub protocol.  |
+| `KZG_COMMITMENT_INCLUSION_PROOF_DEPTH`   | `17`                              | Merkle proof for `blob_kzg_commitments` list item                   |        
```

Comment:
Note that if block body fields count goes beyond 16 this need to be increase, probably it worth increasing it now for the future compatibility sake

Reply:
We can't manipulate merkleization, the depth is what it is. This value will change in future forks when more properties are added. But I can define it better in terms that are fork dependent.

Reply:
we should probably use the new partial container by @etan-status (tangential point from this discussion)

Reply:
[`StableContainer`](https://eips.ethereum.org/EIPS/eip-7495) would be nice, but is unlikely to happen for Deneb given the progress. We can re-visit this for Electra, together with SSZ [transactions](https://eips.ethereum.org/EIPS/eip-6493), [receipts](https://eips.ethereum.org/EIPS/eip-6466), and [withdrawals](https://eips.ethereum.org/EIPS/eip-6465). Worst case, the index will change one time when we transition to `StableContainer` (and also in other places, at the same time)

Reply:
And, if the depth changes in the future, we can also change the size of the proof at that time. just make sure that whenever it is transferred includes `Consensus-Fork` information. on libp2p that is given through forkdigest. on beacon-api it is added via HTTP header, and so on. so it is clear how to parse it. beaacon-API light client endpoints never needed a version bump for this reason because they were designed with forward compatibility in mind from the start.